backend/python/requirements.txt

```diff
@@ -59,3 +63,4 @@ typing-extensions==3.7.4.3
 uritemplate==3.0.1
 urllib3==1.26.4
 Werkzeug==1.0.1
+zipp==3.4.1
```

Comment:
Are `pytest-mock` and `mongomock` required dependencies as well? If so might need to `pip freeze > requirements.txt` again